a10_octavia/controller/worker/tasks/a10_compute_tasks.py

```diff
@@ -124,3 +183,38 @@ def revert(self, result, *args, **kwargs):
                       "still be in use for server group: %(sg)s due to "
                       "error: %(except)s",
                       {'sg': server_group_id, 'except': e})
+
+
+class DeleteStaleCompute(BaseComputeTask):
+    """Delete Stale Compute for Failover"""
+
+    def execute(self, vthunder):
+        if vthunder:
+            try:
+                self.compute.delete(vthunder.compute_id)
+            except Exception as e:
+                LOG.exception("Failed to delete stale compute %s due to: %s",
+                              vthunder.compute_id, str(e))
+                # pass here in case the compute is already deleted
+                pass
+
+
+class FailoverPausedCompute(BaseComputeTask):
+    def __init__(self, **kwargs):
+        super(FailoverPausedCompute, self).__init__(**kwargs)
+        self.vthunder_repo = a10_repo.VThunderRepository()
+
+    def execute(self, vthunder):
+        if vthunder:
+            try:
+                amp, fault = self.compute.get_amphora(vthunder.compute_id)
+                if amp.status == a10constants.COMPUTE_PAUSED:
+                    self.vthunder_repo.set_vthunder_health_state(
+                        db_apis.get_session(), vthunder.id, amp.status)
+                    LOG.info("The vThunder instance is paused, skipping the failover...")
+                    raise a10_exceptions.FailoverOnPausedCompute()
+            except Exception as e:
+                LOG.exception("Failed to find compute %s du to: %s",
+                              vthunder.compute_id, str(e))
+                # pass here in case the compute is already deleted
+                pass
```

Comment:
If we have a log then we don't need a pass.

Reply:
done